packages/url_launcher/url_launcher/lib/url_launcher.dart

```diff
@@ -9,6 +9,10 @@ import 'package:flutter/services.dart';
 import 'package:flutter/widgets.dart';
 import 'package:url_launcher_platform_interface/url_launcher_platform_interface.dart';
 
+export 'src/link.dart' show Link;
+export 'package:url_launcher_platform_interface/link.dart'
+    show FollowLink, LinkTarget, LinkWidgetBuilder;
```

Comment:
(You do need this export though)

Reply:
Another way of going about this, to minimize the exposure of people to the new Link class (it can cause collisions with people's code?) is to have a separate barrel file:

`url_launcher/lib/link.dart` with lines 12, 13, 14 from this pr.

That way people can continue using `url_launcher` pretty much as normal, but if they want the link widget they need to do:

import 'package:url_launcher/link.dart'`

WDYT?

Reply:
I like it!